yadage/workflow_loader.py

```diff
@@ -1,5 +1,15 @@
 import yadageschemas
 
 def workflow(source, toplevel, schema_name='yadage/workflow-schema', schemadir=None, validate=True):
+    '''
+    load (and validate) workflow from source
+    param source: URI fragment to the source
+    param toplevel: base URI to resolve references from
+    schemadir
+    '''
     data = yadageschemas.load(source, toplevel, schema_name, schemadir, validate)
     return data
+
+def validate(data, schema_name='yadage/workflow-schema', schemadir=None):
```

Comment:
We are building the yadage workflow in the client following this [patch](https://github.com/reanahub/reana-client/pull/9/files#diff-670175eb24e7fca435fba316a6b7066fR33). Seems a bit different, is it related with the fact that we are using `yadageschemas==0.6.0`?